lms/envs/devstack.py

```diff
@@ -133,13 +133,17 @@ def should_show_debug_toolbar(_):
 FEATURES['ENABLE_VIDEO_ABSTRACTION_LAYER_API'] = True
 
 ########################## SECURITY #######################
-FEATURES['ENFORCE_PASSWORD_POLICY'] = False
+FEATURES['ENFORCE_PASSWORD_POLICY'] = True
 FEATURES['ENABLE_MAX_FAILED_LOGIN_ATTEMPTS'] = False
 FEATURES['SQUELCH_PII_IN_LOGS'] = False
 FEATURES['PREVENT_CONCURRENT_LOGINS'] = False
 FEATURES['ADVANCED_SECURITY'] = False
-PASSWORD_MIN_LENGTH = None
-PASSWORD_COMPLEXITY = {}
+PASSWORD_MIN_LENGTH = 6
+PASSWORD_COMPLEXITY = {
+    'UPPER': 1,
+    'LOWER': 1,
+    'DIGITS': 1
```

Comment:
Do we need to put the same settings in lms/envs/common.py

Reply:
No, we don't need to do it. It has precedence over common.py.